modules/stanford_soe_helper_page/stanford_soe_helper_page.info

```diff
@@ -29,32 +29,32 @@ features[ctools][] = views:views_default:3.0
 features[ds_field_settings][] = node|stanford_page|full
 features[ds_layout_settings][] = node|stanford_page|full
 features[features_api][] = api:2
+features[field_base][] = field_s_banner_accent_color
 features[field_base][] = field_s_banner_caption_horizonta
 features[field_base][] = field_s_banner_caption_vertical
 features[field_base][] = field_s_banner_image
 features[field_base][] = field_s_banner_link
 features[field_base][] = field_s_top_banner_caption_horiz
 features[field_base][] = field_s_top_banner_caption_vert
+features[field_base][] = field_s_top_banner_color
 features[field_group][] = group_s_page_banner|node|stanford_page|form
 features[field_group][] = group_s_page_top_banner|node|stanford_page|form
 features[field_group][] = group_s_postcard_content|node|stanford_page|full
 features[field_group][] = group_s_postcard_image|node|stanford_page|full
 features[field_group][] = group_s_postcard|node|stanford_page|full
 features[field_instance][] = node-stanford_page-field_s_apachesolr_index
+features[field_instance][] = node-stanford_page-field_s_banner_accent_color
```

Comment:
@sherakama I don't think they want to add these in. I think there will be a conflict with `field_instance`s and `field_base`s in other Features.

Reply:
@jbickar @minorwm @cjwest 

This depends; If this module is adding these fields to the stanford-page type then this is ok. If this module is tracking fields that are in other modules then it is not ok. 

It looks like this is adding to the stanford-page type and not duplicating values. Please confirm that this assumption is correct and that the .info file has a dependency on stanford_page.

Reply:
OK, thanks.